examples/rest_api/mlflow_tracking.py

```diff
@@ -0,0 +1,71 @@
+"""
+This simple example shows how we should use MLFlow Rest API for creating new runs and log parameters/metrics.
```

Comment:
We should explain when you'd want to use it this way here. For example, this might be useful to embed in an application where you don't want to depend on the whole mlflow library, or to make your own HTTP requests in another programming language (not Python)